core/src/main/java/org/mule/runtime/core/processor/strategy/MultiReactorProcessingStrategyFactory.java

```diff
@@ -39,8 +40,9 @@
 public class MultiReactorProcessingStrategyFactory implements ProcessingStrategyFactory {
 
   @Override
-  public ProcessingStrategy create(MuleContext muleContext) {
-    return new MultiReactorProcessingStrategy(() -> muleContext.getSchedulerService().customScheduler("event-loop", 1),
+  public ProcessingStrategy create(MuleContext muleContext, String schedulersNamePrefix) {
+    return new MultiReactorProcessingStrategy(() -> muleContext.getSchedulerService()
+        .customScheduler(schedulersNamePrefix + ".event-loop", 1),
```

Comment:
Should customScheduler now use SchedulerConfig? If it does use it, what do we do with name passed currently?